File: seed/circular-references/src/main/java/com/seed/api/resources/ast/types/PrimitiveValue.java
package com.seed.api.resources.ast.types;

import com.fasterxml.jackson.annotation.JsonValue;

public enum PrimitiveValue {
    STRING("STRING"),

    NUMBER("NUMBER");

    private final String value;

    PrimitiveValue(String value) {
        this.value = value;
    }

    @JsonValue
    @Override
    public String toString() {
        return this.value;
    }
}

File: seed/ruby-sdk-v2/circular-references-advanced/lib/fern_circular_references_advanced/ast/types/primitive_value.rb
# frozen_string_literal: true

module FernCircularReferencesAdvanced
  module Ast
    module Types
      module PrimitiveValue
        extend FernCircularReferencesAdvanced::Internal::Types::Enum

        STRING = "STRING"
        NUMBER = "NUMBER"
      end
    end
  end
end
